### DISCUSSION TITLE
How to temporarily disable argo repository sync?

### DISCUSSION THREAD

**Discussion by mfridman** [argoproj/argo-cd#8801] [Q&A]
Is there a toggle-able way to disable the repository sync?

I.e., I have 3 argo instances all pulling from a common repo. I'd like to log in to 2 instances and temporarily disable sync while testing out a changeset. Once confirmed working I'd like to enable argo sync and have whatever the latest state applied to the other 2 clusters.

In today's world I could disconnect the repository in argo. But this is a lot of effort, so wondering if I'm missing something obvious?

**Comment by chetan-rns** [comment]
Hi @mfridman, does sync windows solve your use case? Please check https://argo-cd.readthedocs.io/en/stable/user-guide/sync_windows/
